### ISSUE TITLE
Add placeholder property to the Chat component

### ISSUE THREAD

**Issue by Xizario** [telerik/kendo-react#150]
The `Type a message...` placeholder and `Send...` aria-label of the chat should be localizable true the LocalizationProvider

**Comment by Xizario** [comment]
After initial review it seems that it would be better for the `placeholder` to be a property, rather than string from the messages. Since even for the english it is expected the user to want to change it from 'Type a message...' to something else.
